evergreen/run-tests.sh

```diff
@@ -63,6 +60,10 @@ else
 fi
 
 echo "Final MongoDB_URI: $MONGODB_URI"
-echo "Final MongoDB URI with multiple mongoses: $MONGODB_URI_WITH_MULTIPLE_MONGOSES"
-for var in TMP TEMP NUGET_PACKAGES NUGET_HTTP_CACHE_PATH APPDATA; do setx $var z:\\data\\tmp; export $var=z:\\data\\tmp; done
```

Comment:
I removed ``setx`` because it was not needed on Cygwin bash  (as evidenced by evergreen going green with this change enabled). Removing the ``setx`` also has the added benefit of allowing the script to run on bash within my on Windows Subsystem for Linux environment, making it a bit easier for me to test changes to this script locally.

Reply:
Acknowledged